app/models/unread_notifications_status.rb

```diff
@@ -2,10 +2,12 @@ class UnreadNotificationsStatus
     attr_accessor :count
     attr_accessor :user
     attr_accessor :parent
+    attr_accessor :unread_journals
```

Comment:
Aceasta pare nefolosit. Este uitat? Daca da trebuie sters. 
Manual:  https://redmine.xops-online.com/issues/34798#ShTraining-Git-03-Pre-commit-inspection-review-each-change-Staging-view-and-index

Reply:
Da, intradevar. L-am eliminat.